.github/workflows/grid-backend_tests.yml

```diff
@@ -7,11 +7,9 @@ on:
       - dev
       - main
       - "0.6.0"
-    paths:
-      - ".github/workflows/grid-*.yml"
-      - "packages/grid/**.py"
-      - "packages/grid/**.txt"
-      - "packages/grid/**.toml"
```

Comment:
Q: Should we also include `` .txt and .toml to file-filters`` workflow as it is inlcuded here.

Reply:
Right @rasswanth-s thanks for the catch! 💚